deploy/sre-prometheus/management-cluster/100-dynatrace-workloads-monitoring.PrometheusRule.yaml

```
@@ -0,0 +1,90 @@
+apiVersion: monitoring.rhobs/v1
```

I think this needs to be created for CAMO Or do we want this in OBO?
```suggestion
apiVersion: monitoring.coreos.com/v1
```

---

The `hypershift_hostedclusters` is in OBO, so I guess we cannot do the same for CAMO 
cc @ravitri